library/ns1_zone.py

```diff
@@ -480,28 +480,28 @@ def diff_in_secondaries(self, have_secondaries, want_secondaries):
         elif len(want_secondaries) != len(have_secondaries):
             return True
 
-        have_hash = self.convert_secondaries_to_hash(have_secondaries)
-        want_hash = self.convert_secondaries_to_hash(want_secondaries)
-        diff = self.diff_params(have_hash, want_hash)
+        have = self.convert_secondaries_to_dict(have_secondaries)
+        want = self.convert_secondaries_to_dict(want_secondaries)
+        diff = self.diff_params(have, want)
         if diff:
             return True
 
         return False
 
-    def convert_secondaries_to_hash(self, secondaries):
-        """Converts a secondaries list to hash. Keys are a concatenated string
+    def convert_secondaries_to_dict(self, secondaries):
+        """Converts a secondaries list to a dictionary. Keys are a concatenated string
         of IP and Port fields.
 
         :param secondaries: List of secondary dicts
         :type secondaries: list
-        :return: Hash of secondary dicts
+        :return: Dict of secondary dicts
         :rtype: dict
         """
-        secondaries_hash = {}
+        secondaries_dict = {}
         for secondary in secondaries:
             socket = "{0}:{1}".format(secondary["ip"], secondary["port"])
-            secondaries_hash[socket] = secondary
-        return secondaries_hash
+            secondaries_dict[socket] = secondary
+        return secondaries_dict
```

Comment:
could make the keys tuples - little more crispy than catenating

Reply:
🤦‍♂ tuples as keys, why have I never done this before?